Examples/Algorithms/Geant4/src/ParticleTrackingAction.cpp

```diff
@@ -24,7 +24,7 @@ ActsExamples::ParticleTrackingAction::ParticleTrackingAction(
 
 void ActsExamples::ParticleTrackingAction::PreUserTrackingAction(
     const G4Track* aTrack) {
-  auto& eventData = EventStoreRegistry::eventData();
+  auto& eventData = *m_cfg.eventStore;
```

Comment:
*In principle* we could add a member function returning a reference to eventData and then just add `()` to the places where this is used... this would be nice, but is of course not necessary

Of course this holds for all classes with similar interface.